src/seraphis_core/jamtis_destination.h

```diff
@@ -54,12 +54,14 @@ namespace jamtis
 ///
 struct JamtisDestinationV1 final
 {
-    /// K_1 = k^j_g G + k^j_x X + k^j_u U + K_s   (address spend key)
-    rct::key addr_K1;
-    /// xK_2 = xk^j_a xK_fr                       (address view key)
-    crypto::x25519_pubkey addr_K2;
-    /// xK_3 = xk^j_a xK_ua                       (DH base key)
-    crypto::x25519_pubkey addr_K3;
+    /// K^j_s     = k^j_g G + k^j_x X + k^j_u U + K_s (address spend key)
+    rct::key addr_s;
```

Comment:
```suggestion
    rct::key addr_Ks;
```

Lowercase letter implies private key or scalar.